LiskHQ/lisk-desktop
# Fix general (style) problems - Closes #430

### What did I fix?
- Prevented passphrase input from jumping up and down when error is displayed/not displayed
- Changed "Latest search" -> "Last searches"
- Cleared transactions after account switched
- Improved account switcher for mobile
- Removed too big space between columns in transaction dashboard
- Made all box paddings and header consistent on XL, L, M
- Made account visual size appropriate for mac tablet

### How to test it?
- Try to make a transaction & check that the passphrase input fields are not jumping up on error
- Make sure transactions are cleared after account switch (means, you should no longer see the old transactions in the list before the new ones load)
- Check account switcher on mobile
- Check spacing in tx dashboard
- Check the box paddings and h2 font sizes on XL, L and M and make sure they are always consistent
- Check account visual in header on mac tablet

### Review checklist
- The PR solves #430
- All new code is covered with unit tests
- All new features are covered with e2e tests
- All new code follows best practices
